scenario/context.py

```diff
@@ -508,7 +533,7 @@ def __init__(
         self.juju_log: List["JujuLogLine"] = []
         self.app_status_history: List["_EntityStatus"] = []
         self.unit_status_history: List["_EntityStatus"] = []
-        self.exec_history: Dict[str, List[ops.testing.ExecArgs]] = {}
+        self.exec_history: Dict[str, List[ExecArgs]] = {}
```

Comment:
I guess this is a breaking change, but probably won't effect anyone in practice?

Reply:
Yes, technically. If you were type annotating with `ops.testing.ExecArgs` then the checker would complain, and `repr(ExecArgs)`, `ExecArgs.__class__` and so on are different. I agree that it would be extremely unlikely to effect anyone in practice, particularly since this attribute was only added to Scenario in 7.0.